sky/cli.py

```diff
@@ -4921,6 +4931,43 @@ def local_down():
             f'{colorama.Fore.GREEN}Local cluster removed.{style.RESET_ALL}')
 
 
+def change_config_port():
+    """Set the `kubernetes:ports` to be `ingress` and backup the original value."""
+    
+    if not skypilot_config.loaded_config_path():
+        logger.debug(f'Making base {CONFIG_PATH} file as it did not previously exist.')
+        default_config = {'kubernetes': {'ports': 'loadbalancer'}}
+        config_path = os.path.expanduser(CONFIG_PATH)
+        common_utils.dump_yaml(config_path, default_config)
+        skypilot_config._try_load_config()
+    
+    logger.debug(f'Creating backup at {BACKUP_PATH} of current kubernetes port forwarding option.')
+    backup_path = os.path.expanduser(BACKUP_PATH)
+    backup_port_config = skypilot_config.to_dict()['kubernetes']['ports']
+    common_utils.dump_yaml(backup_path, {'kubernetes': {'ports': backup_port_config}})
+    
+    current_path = skypilot_config.loaded_config_path()
+    logger.debug(f'Set current port forwarding option to ingress in {current_path}.')
+    updated_config = skypilot_config.set_nested(('kubernetes', 'ports'), 'ingress')
+    common_utils.dump_yaml(current_path, updated_config)
+    skypilot_config._try_load_config()
```

Comment:
Can we move this logic to `create_cluster.sh` and `delete_cluster.sh` and do this backup/restore in the shell scripts? Goal is to keep `create_cluster.sh`and `delete_cluster.sh` self-contained and minimize the objective-specific logic in `cli.py`

Reply:
If it's more readable, feel free to create and invoke in-line python script (e.g., with `python -c "...."`)  in `create_cluster.sh` and `delete_cluster.sh`.